packages/docusaurus-theme-classic/src/theme-classic.d.ts

```diff
@@ -261,10 +261,32 @@ declare module '@theme/DocCardList' {
   export default function DocCardList(props: Props): JSX.Element;
 }
 
-declare module '@theme/DocItemFooter' {
-  import type {Props} from '@theme/DocItem';
+declare module '@theme/DocItem/Layout' {
+  export default function DocItemLayout(): JSX.Element;
+}
+
+declare module '@theme/DocItem/Metadata' {
+  export default function DocItemMetadata(): JSX.Element;
+}
 
-  export default function DocItemFooter(props: Props): JSX.Element;
+declare module '@theme/DocItem/Content' {
+  export default function DocItemContent(): JSX.Element;
+}
+
+declare module '@theme/DocItem/TOC/Mobile' {
+  export default function DocItemTOCMobile(): JSX.Element;
+}
+
+declare module '@theme/DocItem/TOC/Desktop' {
+  export default function DocItemTOCDesktop(): JSX.Element;
+}
+
+declare module '@theme/DocItem/Paginator' {
+  export default function DocItemPaginator(): JSX.Element;
+}
+
+declare module '@theme/DocItemFooter' {
```

Comment:
Since we are making breaking changes anyway, can we make this `DocItem/Footer`?

Reply:
Yes 👍 also thought about it